graylog2-server/src/main/java/org/graylog2/shared/plugins/ChainingClassLoader.java

```diff
@@ -121,4 +121,8 @@ public InputStream getResourceAsStream(String name) {
     public boolean addClassLoader(ClassLoader classLoader) {
         return classLoaders.add(classLoader);
     }
+
+    public Class<?> classForName(String className) throws ClassNotFoundException {
```

Comment:
Is this method really necessary? Wouldn't a call to [`ClassLoader#loadClass(String)`](https://docs.oracle.com/javase/8/docs/api/java/lang/ClassLoader.html#loadClass-java.lang.String-) suffice?